tagstudio/src/qt/widgets/preview_panel.py

```diff
@@ -407,6 +408,10 @@ def update_widgets(self):
 
                     except (UnidentifiedImageError, FileNotFoundError, cv2.error):
                         pass
+                    except DecompressionBombError as e:
+                        logging.info(
```

Comment:
maybe these except blocks can be merged together, what do you think?
```suggestion
                    except (DecompressionBombError, UnidentifiedImageError, FileNotFoundError, cv2.error) as e:
                        logging.exception(
```

Reply:
Probably smart yep